op.c

```diff
@@ -12227,7 +12227,6 @@ Perl_ck_eval(pTHX_ OP *o)
         /* Store a copy of %^H that pp_entereval can pick up. */
         HV *hh = hv_copy_hints_hv(GvHV(PL_hintgv));
         OP *hhop;
-        STOREFEATUREBITSHH(hh);
```

Comment:
Hmm, took me a little while to work out why this bit, until I saw that we're now reconstructing the feature bits from the entire hinthash. Is that OK? In general the hh might contain a lot more keys (e.g. other bits of enabled syntax and options), so iterating the whole thing might take a little while, as compared just storing/fetching the UV as we used to.